packages/picasso/src/RatingThumbs/RatingThumbs.tsx

```diff
@@ -0,0 +1,139 @@
+import { makeStyles, Theme } from '@material-ui/core/styles'
+import { BaseProps, SizeType } from '@toptal/picasso-shared'
+import classNames from 'classnames'
+import React, { ChangeEvent, forwardRef, useCallback, useState } from 'react'
+
+import { Container } from '..'
+import { ThumbsDown16, ThumbsDown24, ThumbsUp24 } from '../Icon'
+import ThumbsUp16 from '../Icon/ThumbsUp16'
```

Comment:
```suggestion
import Container from '../Container'
import { ThumbsDown16, ThumbsDown24, ThumbsUp16, ThumbsUp24 } from '../Icon'
```
For consistency :+1:

Reply:
I will update it